organizations/urls.py

```diff
@@ -1,3 +1,9 @@
 """
-urls.py -- useful some day when views.py comes alive
+URLS for organizations
 """
+from django.conf.urls import url, include
+
+# pylint: disable=invalid-name
+urlpatterns = [
+    url(r'^v0/', include('organizations.v0.urls', namespace='v0')),
```

Comment:
As this API has not passed the arch council therefore version is v0. Once it does it will updated to v1.